### ISSUE TITLE
RunWorkflowWidget is broken because of removed ert_workflow_list

### ISSUE THREAD

**Issue by eivindjahren** [equinor/ert#4727]
In commit a6aa00c6a4f9043cd29c8bdcd7b08907ab068ad8 `getWorkflowList()` was removed. This was used by `RunWorkflowWidget`, but because its tests uses a mocked object, we didn't notice that it broke: https://github.com/equinor/ert/blob/ecf11cf8b9471c2cdd4ce18f46151b6df82ca2cb/tests/unit_tests/gui/test_gui_load.py#L23-25 .
We need to at least fix RunWorkflowWidget. It uses `ert.getWorkflowList().getWorkflowNames()` to get the list of workflow names. The way that is done now is by `list(ert.resConfig().workflows.keys())`. The mock needs to be updated accordingly.
Once this is done we should create another issue to remove the mock, and use synthetic test data instead.

**Comment by Jehan-h20220012** [comment]
Can I work on this?

**Comment by eivindjahren** [comment]
@Jehan-h20220012 This issue is probably a bit complicated for a newcomer, and we will fix it fairly quickly. I have asked around for a good first issue, and will get back to you :)

**Comment by oyvindeide** [comment]
@Jehan-h20220012 thanks for your interest in solving this! As @eivindjahren mentioned, this issue is a bit time sensitive, so we would need some close follow up. But if you would like to give it a try we really appreciate it! We have full understanding if that is a bit daunting though, and will be happy to come up with an alternative issue if you would like to contribute.

**Comment by Jehan-h20220012** [comment]
Sure @eivindjahren @oyvindeide I am ready to work on any issue for begineers and will fulfil whatever is to be done to merge my PR and close the issue. Please do let me know if you find some issue suitable for me.

**Comment by eivindjahren** [comment]
@Jehan-h20220012 Feel free to work on this issue. I have updated the issue description to help you out :)

**Comment by Jehan-h20220012** [comment]
Thank You so much @eivindjahren

**Comment by Jehan-h20220012** [comment]
> > Sure @eivindjahren @oyvindeide I am ready to work on any issue for begineers and will fulfil whatever is to be done to merge my PR and close the issue. Please do let me know if you find some issue suitable for me.
> 
> Awesome! As @eivindjahren mentions, this issue is available to work on, otherwise we have a `good-first-issue` label. I must admit that the list of `good-first-issue` has not been very well curated, so I have started to go through our issue list to identify more issues that are appropriate, so more issues will get that tag going forward.

Thank You so much. I will look into it and start working asap to raise PR to merge for solving the issue

**Comment by Jehan-h20220012** [comment]
Hello @oyvindeide @eivindjahren Can I connect with you guys via slack or something? It will help me solve the issue

**Comment by oyvindeide** [comment]
> Hello @oyvindeide @eivindjahren Can I connect with you guys via slack or something? It will help me solve the issue

Hi, sorry for being late to reply. Sadly our organization does not have any public slack channels, so we will mainly communicate though github. But feel free to open a draft PR and we can discuss there, or ask questions here.

**Comment by Jehan-h20220012** [comment]
Okay no issue!
@oyvindeide

**Comment by oysteoh** [comment]
We probably just need to fix this issue now as it is a bit urgent. There are more bugs added to the board labeled "good first issue" - i suggest picking up these instead.

**Comment by Jehan-h20220012** [comment]
@oysteoh I am readily keen to work on this. Its just that I need some help.
Do you know how to come up on a solution to this problem?
If you guide me with certain steps I will work on it and surely raise a PR ASAP.

**Comment by Jehan-h20220012** [comment]
Respected @oysteoh ,
If you have the solution please help me with it so that I will raise a PR to merge and also I am trying to fix this issue and understanding the stuff. If there is someone working on this they can also raise a PR so meanwhile, please do let me work.
I understand the time constrains and I am ready to work accordingly whatever you say. Is there someone else working on it and already have the solution?

**Comment by Jehan-h20220012** [comment]
@oysteoh @eivindjahren @oyvindeide Please consider and approve my PR and let it get merged as I am working on this since past few days and I have to show this in my acedemics as an assignment.
So I request you to let me raise the PR and get it merged. Hope you understand and consider it!
